Components/modal.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Flame UI | Text-Utility</title>
+    <script src="https://kit.fontawesome.com/d3026d29a6.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../gist.css">
+    <link rel="stylesheet" href="../styles.css">
+    <link rel="stylesheet" href="../components.css">
+    <link rel="stylesheet" href="./modal.css">
+</head>
+
+<body>
+    <div class="nav-container">
+        <h1 class="nav-logo">Flame UI</h1>
+        <div class="nav-links">
+            <a class="link" href="../index.html">Home</a>
+            <a class="link link-active" href="./avatar.html">Documentation</a>
+            <a class="link" href="https://github.com/j24m"><svg xmlns="http://www.w3.org/2000/svg"
+                    xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" role="img"
+                    class="iconify iconify--uil" width="32" height="32" preserveAspectRatio="xMidYMid meet"
+                    viewBox="0 0 24 24">
+                    <path
+                        d="M12 2.247a10 10 0 0 0-3.162 19.487c.5.088.687-.212.687-.475c0-.237-.012-1.025-.012-1.862c-2.513.462-3.163-.613-3.363-1.175a3.636 3.636 0 0 0-1.025-1.413c-.35-.187-.85-.65-.013-.662a2.001 2.001 0 0 1 1.538 1.025a2.137 2.137 0 0 0 2.912.825a2.104 2.104 0 0 1 .638-1.338c-2.225-.25-4.55-1.112-4.55-4.937a3.892 3.892 0 0 1 1.025-2.688a3.594 3.594 0 0 1 .1-2.65s.837-.262 2.75 1.025a9.427 9.427 0 0 1 5 0c1.912-1.3 2.75-1.025 2.75-1.025a3.593 3.593 0 0 1 .1 2.65a3.869 3.869 0 0 1 1.025 2.688c0 3.837-2.338 4.687-4.563 4.937a2.368 2.368 0 0 1 .675 1.85c0 1.338-.012 2.413-.012 2.75c0 .263.187.575.687.475A10.005 10.005 0 0 0 12 2.247z"
+                        fill="currentColor"></path>
+                </svg></a>
+        </div>
+    </div>
+
+    <div class="sidebar-docs-container">
+        <div class="side-bar">
+            <h3 class="sidebar-cmp-heading">Quick Installation</h3>
+            <ul class="ul">
+                <a class="link" href="./getstarted.html">
+                    <li>Get Started</li>
+                </a>
+                <h3 class="sidebar-cmp-heading">Components</h3>
+                <ul class="ul">
+                    <a class="link" href="./avatar.html">
+                        <li>Avatar</li>
+                    </a>
```

Comment:
it should be ul --> li --> a

Reply:
Will keep this in mind and make the changes later as this particular change will have to be done in all the files. Thanks for mentioning this, nobody noticed. :)